portfolio/src/main/java/com/google/sps/data/Comments.java

```diff
@@ -0,0 +1,74 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+import java.util.Map;
+import java.util.Map.Entry;
+import java.util.HashMap;
+// import java.time.format.DateTimeFormatter;
+
+/**
+ * Class representing the comments left on our webpage, not threadsafe.
```

Comment:
HashMap itself is thread safe. If you think the class is not thread-safe, it is better to add a clock protecting the threads.